sdk/internal/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+Support test recording assets external to repository
```

Comment:
We should bump the minor version here and in the [version.go](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/internal/version.go#L14) file.

Reply:
@jhendrixMSFT should that be done in this PR or as part of a separate release process?

Reply:
True we can wait for that.